README.md

```diff
@@ -1,27 +1,59 @@
 [<img src="https://devforum.okta.com/uploads/oktadev/original/1X/bf54a16b5fda189e4ad2706fb57cbb7a1e5b8deb.png" align="right" width="256px"/>](https://devforum.okta.com/)
 [![Maven Central](https://img.shields.io/maven-central/v/com.okta.authn.sdk/okta-authn-sdk-api.svg)](https://search.maven.org/#search%7Cga%7C1%7Cg%3A%22com.okta.authn.sdk%22%20a%3A%22okta-authn-sdk-api%22)
 [![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
-[![Support](https://img.shields.io/badge/support-Developer%20Forum-blue.svg)](https://devforum.okta.com/)
+[![Support](https://img.shields.io/badge/support-Developer%20Forum-blue.svg)][devforum]
+[![API Reference](https://img.shields.io/badge/docs-reference-lightgrey.svg)][javadocs]
 
-# Okta Authentication SDK
+# Okta Java Authentication SDK
 
+> :warning: Beta alert! This library is in beta. See [release status](#release-status) for more information.
+
+* [Release status](#release-status)
+* [Need help?](#need-help)
+* [Getting started](#getting-started)
+* [Usage guide](#usage-guide)
+* [Configuration reference](#configuration-reference)
+* [Building the SDK](#building-the-sdk)
+* [Contributing](#contributing)
+ 
 The Okta Authentication SDK is a convenience wrapper around [Okta's Authentication API](https://developer.okta.com/docs/api/resources/authn.html).
 
 **NOTE:** Using an OAuth 2.0 or OIDC to integrate your application instead of this library will require much less work, and has a smaller risk profile. Please see [this guide](https://developer.okta.com/use_cases/authentication/) to see if using this API is right for your use case. You could also use our [Spring Boot Integration](https://github.com/okta/okta-spring-boot), or [Spring Security](https://developer.okta.com/blog/2017/12/18/spring-security-5-oidc) out of the box.
```

Comment:
nitpick: remove **an**

Reply:
Let's always spell out "OpenID Connect". Here's a suggested rewrite with a few improvements:

    Using OpenID Connect to add authentication to your application (instead of this library) is easier and less complicated. In most cases, you don't need to custom-build an authentication interaction with this library.

    See [this guide](https://developer.okta.com/use_cases/authentication/) to see which approach is best for your use case. You can use our [Spring Boot integration](https://github.com/okta/okta-spring-boot), or [Spring Security](https://developer.okta.com/blog/2017/12/18/spring-security-5-oidc) out of the box.`

Reply:
I like the suggestion, but this part:
> you don't need to custom-build an authentication interaction with this library.
Seems like it is talking about the authn lib instead of the OIDC one